rmf_site_editor/src/lib.rs

```diff
@@ -146,6 +148,8 @@ pub fn run(command_line_args: Vec<String>) {
     {
         app.add_plugins(
             DefaultPlugins
+                .build()
+                .disable::<LogPlugin>()
```

Comment:
This is a bit of an unfortunate necessity since the snippet below:

```
                .set(LogPlugin {
                    filter: "bevy_asset=error,wgpu=error".to_string(),
                    ..default()
                })
```

Allows us to filter the log based on crates and severity, since the plugin was disabled this is not possible anymore.

I guess we can either:
* If this can be reverted, keep the logging plugin and its filtering (I assume not possible based on what you said)
* If this cannot be reverted, either remove the snippet below since it doesn't do anything now or try to get some similar configuration into the current logging plugin.

Reply:
Just saw the filter is available in the `console.rs` file, then we can just copy the updated filter below into the `default_filter` variable and remove it from here